libvast/test/pattern.cpp

```diff
@@ -22,6 +22,7 @@
 
 using namespace vast;
 using namespace std::string_literals;
+using namespace std::string_view_literals;
```

Comment:
I don't spot any `""sv`, is this really used?

Reply:
✅